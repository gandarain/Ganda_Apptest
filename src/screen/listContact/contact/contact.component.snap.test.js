import React from 'react'
import renderer from 'react-test-renderer'

import { mockContact } from '../../../fixture'

import Contact from './contact.component'

jest
  .mock('@react-navigation/native', () => ({
    useNavigation: () => ({
      navigation: () => ({
        navigate: jest.fn()
      })
    })
  }))
  .mock('react-native-vector-icons/MaterialCommunityIcons', () => 'Icon')

describe('Contact Component', () => {
  it('should render contact component', () => {
    const rendered = renderer.create(<Contact item={mockContact.data.data} />)
    const tree = rendered.toJSON()

    expect(tree).toMatchSnapshot()
  })
})
