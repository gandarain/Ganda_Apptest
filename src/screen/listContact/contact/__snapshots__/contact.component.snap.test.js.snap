// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Contact Component should render contact component 1`] = `
<View
  accessibilityLabel="ContainerItem"
  accessible={true}
  collapsable={false}
  focusable={true}
  onClick={[Function]}
  onResponderGrant={[Function]}
  onResponderMove={[Function]}
  onResponderRelease={[Function]}
  onResponderTerminate={[Function]}
  onResponderTerminationRequest={[Function]}
  onStartShouldSetResponder={[Function]}
  style={
    {
      "alignItems": "center",
      "backgroundColor": "#FFFFFF",
      "borderRadius": 8,
      "elevation": 2,
      "flexDirection": "row",
      "justifyContent": "space-between",
      "marginHorizontal": 15,
      "marginVertical": 15,
      "opacity": 1,
      "padding": 15,
      "shadowColor": "#000",
      "shadowOffset": {
        "height": 3,
        "width": 0,
      },
      "shadowOpacity": 0.1,
      "shadowRadius": 2,
    }
  }
>
  <View
    accessibilityLabel="ContentItem"
    style={
      {
        "alignItems": "center",
        "flexDirection": "row",
        "justifyContent": "flex-start",
      }
    }
  >
    <Image
      accessibilityLabel="ImageItem"
      source={
        {
          "uri": "http://vignette1.wikia.nocookie.net/lotr/images/6/68/Bilbo_baggins.jpg/revision/latest?cb=20130202022550",
        }
      }
      style={
        {
          "borderRadius": 50,
          "height": 50,
          "width": 50,
        }
      }
    />
    <View
      accessibilityLabel="SubContentItem"
      style={
        {
          "marginLeft": 15,
        }
      }
    >
      <Text
        accessibilityLabel="TextFullName"
        style={
          {
            "color": "#5E5E5E",
            "fontFamily": "Roboto-Bold",
            "fontSize": 18,
          }
        }
      >
        Bilbo
         
        Baggins
      </Text>
      <Text
        accessibilityLabel="AgeText"
        style={
          {
            "color": "#5E5E5E",
            "fontFamily": "Roboto-Regular",
            "fontSize": 18,
          }
        }
      >
        111
      </Text>
    </View>
  </View>
  <Icon
    accessibilityLabel="IconDetail"
    color="#FF6049"
    name="chevron-right"
    size={25}
  />
</View>
`;
